src/auth-service/controllers/create-user.js

```diff
@@ -1380,6 +1380,7 @@ const createUser = {
   },
   loginInViaEmail: async (req, res, next) => {
     try {
+      console.log("Inside Controller")
```

Comment:
Hi @BenjaminSsempala , please remove this console log.

- In our codebase, we usually use (require) the local custom one `logObject()` for logging because it is configured NOT to work in production
- EXPLANATION: It is considered a security risk to log things in production. It can even sometimes lead to performance issues in the long run.

Reply:
Thanks For Catching this @Baalmart 
I've fixed it